docs/api/app.md

```diff
@@ -1427,11 +1427,26 @@ This is the user agent that will be used when no user agent is set at the
 app has the same user agent.  Set to a custom value as early as possible
 in your app's initialization to ensure that your overridden value is used.
 
-### `app.runningUnderRosettaTranslation` _macOS_ _Readonly_
+### `app.runningUnderRosettaTranslation` _macOS_ _Readonly_ _Deprecated_
```

Comment:
Can you add depreciation warning for this property? You can find a few examples searching `EmitWarning` in the codebase.